dev/sparktestsupport/modules.py

```diff
@@ -1126,6 +1126,7 @@ def __hash__(self):
         "pyspark.ml.tests.connect.test_parity_clustering",
         "pyspark.ml.tests.connect.test_parity_evaluation",
         "pyspark.ml.tests.connect.test_parity_feature",
+        "pyspark.ml.tests.connect.test_parity_pipeline",
```

Comment:
Seems like this hangs in https://github.com/apache/spark/actions/runs/15810775399